app/services/ProductDAO.scala

```diff
@@ -59,30 +67,44 @@ trait ProductDAO {
 
   /** Update statement into the database to update a product given in the product instance using its UUID
    *
-   * @param id
-   * @param product
-   * @tparam T
+   * @param id: String of the UUID given by the user.
+   * @param product: Given product instance to use in query.
+   * @tparam T: What the query will put its data in the format of.
    * @return
    */
   def updateResultById[T](id: String, product: Products): Int = {
-    sql"update Product set name = ${product.name}, quantity = ${product.quantity}, price = ${product.price} where id=${id}::uuid "
+    val status = Try(
+      sql"update Product set name = ${product.name}, quantity = ${product.quantity}, price = ${product.price} where id=$id::uuid "
       .update
       .run
       .transact(xa)
       .unsafeRunSync()
+    )
+
+    status match {
+      case Success(num) => num
+      case Failure(_) => -1
+    }
   }
 
   /** Delete statement into the database to delete a product given its UUID
    *
-   * @param id
-   * @tparam T
+   * @param id: String of the UUID given by the user.
+   * @tparam T: What the query will put its data in the format of.
    * @return
    */
   def deleteResultById[T](id: String): Int = {
-    sql"delete from Product where id=${id}::uuid "
-      .update
-      .run
-      .transact(xa)
-      .unsafeRunSync()
+    val status = Try(
+      sql"delete from Product where id=$id::uuid "
+        .update
+        .run
+        .transact(xa)
+        .unsafeRunSync()
+    )
+
+    status match{
```

Comment:
formatting needs fixing

Reply:
Sorry, I don't understand what the issue is

Reply:
should be `status match {` (space between brace and keyword)

Reply:
Ah, I see, I'll make sure to fix it!